src/plugins/intel_cpu/src/transformations/cpu_opset/common/pass/decompose_rms_norm.cpp

```diff
@@ -46,9 +47,10 @@ DecomposeRMSNorm::DecomposeRMSNorm() {
         auto eps = ov::op::v0::Constant::create(data_precision, {}, {node->get_epsilon()});
         auto add_eps = std::make_shared<ov::op::v1::Add>(mean, eps);
         auto sqrt = std::make_shared<ov::op::v0::Sqrt>(add_eps);
-        auto div_const = ov::op::v0::Constant::create(data_precision, {}, {-1});
-        auto div = std::make_shared<ov::op::v1::Power>(sqrt, div_const);
-        auto mul1 = std::make_shared<ov::op::v1::Multiply>(data, div);
+        // auto div_const = ov::op::v0::Constant::create(data_precision, {}, {-1});
```

Comment:
Please remove commented code and unrequired include

Reply:
Okay I see, but this commit is just for checking CI failure in CPU.